internal/commands/sbomtest/sbomtest.go

```diff
@@ -101,7 +101,12 @@ func TestWorkflow(
 		ct = MIMETypeText
 	}
 
-	return []workflow.Data{workflowData(buf.Bytes(), ct)}, err
+	summaryData, summaryContentType, err := BuildTestSummary(results.Summary)
+	if err != nil {
+		return nil, err
```

Comment:
can we use the error factory here? ideally the user would see a user-friendly error message.

Reply:
good point! should we do a specific error, or would a `NewInternalError` work? also while we're at it, should we also update https://github.com/snyk/cli-extension-sbom/blob/9347cb700fdca8fd37af703147ccc28cea66c973/internal/snykclient/helpers.go#L32

Reply:
`NewInternalError` sounds fine! 🙂 

the other one i cannot follow, isn't this being handled by the workflow function? i'd like it if we can change it in a separate PR, if it turns out there's still a bug.

Reply:
in that case the `parseResponse` function returns the raw unmarshal error to the user and it's not wrapped anywhere in the call chain (workflow -> snykclient -> parseResponse)